pyam/utils.py

```diff
@@ -129,6 +129,14 @@ def format_data(df):
         missing = list(set(IAMC_IDX) - set(df.columns))
         raise ValueError("missing required columns `{}`!".format(missing))
 
+    if 'notes' in df.columns:
+        logger().info('Ignoring notes column in dataframe')
+        df.drop(columns='notes', inplace=True)
+        df = df[
+            (~df.model.str.contains('SSP Public Database'))
```

Comment:
Sorry for raising this again, but you shouldn't need checking both the database name and the `@` - just checking for the `@` should be enough.

Reply:
No need to apologise, I wasn't thinking (long day)